app/src/main/java/com/github/na206851/Task_6.java

```diff
@@ -0,0 +1,52 @@
+package com.github.na206851;
+
+public class Task_6 {
+
+    public int expressionA(int n) {
+        int a = 1;
+        return a << n;
+    }
+
+    public int expressionB(int n, int m) {
+        int a = 1;
+        return (a << n) + (a << m);
+    }
+
+    public int expressionC(int a, int n) {
+        return a & createMask(n);
+    }
+
+    private int createMask(int n) {
```

Comment:
Check convention, how methods should come in class according to their visibility